### PR TITLE
feat: track events with audit logs

### LINKED ISSUE #90: Start tracking audits/events
Please note that these are going to be tracked on `audit` table. We already have a lib called `audit` for this. One additional column we need on this audit table is `summary`, which will have a human friendly message

Eg: 
`John Doe invited Will Smith as an admin`
`Will Smith accepted an invitation to join Company X`

Here are few places we can add these events/logs to:
* Signup
* Company create
* Stakeholder invite
* Stakeholder re-invite
* Stakeholder accept
* Stakeholder update
* Stakeholder remove

### PR DESCRIPTION
**Pull request by G3root**
fixes #90

### CODE REVIEW

**Comment by G3root** [comment]
known issues 

- `stakeholder.invite` and `stakeholder.remove` has invitee's name to be undefined should be fixed after #79

**Comment by dahal** [comment]
> known issues
> 
> * `stakeholder.invite` action and `stakeholder.remove` action while revoking invite has undefined value. it should be fixed after [Improve stakeholder invite flow #79](https://github.com/opencapco/opencap.co/issues/79)

@G3root I don't think #79 will fix it. Lets create an issue for this.

File: src/server/audit/actions.ts [hunk 1]
```diff
@@ -1,4 +1,14 @@
...
+  company: ["create"],
+  stakeholder: [
+    "invite",
+    "re-invite",
+    "accept",
+    "update",
+    "remove",
+    "revoke-invite",
+    "deactivate",
+    "activate",
```
Comment (dahal) [inline src/server/audit/actions.ts:12]: Let's keep these events/actions consistent, lets go with `signed-up`, `invited`, `re-invited`, `accepted`. 

Note: these events has already occurred, it only makes sense to stick with past tense.

**Review by dahal — CHANGES_REQUESTED** [review]
Everything looks great @G3root , I have left a comment, else its good to go.

**Pull request merged**